extension/background.ts

```diff
@@ -124,7 +128,7 @@ function initializeConfigFromCloudOrLocalStorageOrDefaults(cloudStorage) {
   const kanjiInfoLabelList = RcxDict.prototype.kanjiInfoLabelList;
   for (i = 0; i < kanjiInfoLabelList.length; i += 2) {
     const kanjiInfoKey = kanjiInfoLabelList[i];
-    if (cloudStorage.kanjiInfo && cloudStorage.kanjiInfo[kanjiInfoKey]) {
+    if (cloudStorage.kanjiInfo) {
```

Comment:
Oh, this is clever. Since `kanjiInfo` always initialized as a whole we don't need to check the individual values existing!

👍🏾

Reply:
Thanks!